content/guides/references/proxy-configuration.md

```diff
@@ -117,6 +117,40 @@ If an uppercase and a lowercase version of the proxy settings are supplied (for
 example, `HTTP_PROXY` and `http_proxy` are both set), the lowercase variable
 will be preferred.
 
+## Using a custom CA
```

Comment:
```suggestion
## Using a custom certificate authority (CA)
```

Figure we should throw the phrase "certificate authority" in here for searchability. `advanced-installation.md` could use a similar change

Reply:
Good call. Updated in both places.